common/config/rush/experiments.json

```diff
@@ -18,7 +18,13 @@
    * By default, rush passes --no-prefer-frozen-lockfile to 'pnpm install'.
    * Set this option to true to pass '--frozen-lockfile' instead.
    */
-  // "usePnpmFrozenLockfileForRushInstall": true,
+  "usePnpmFrozenLockfileForRushInstall": true,
+
+  /**
+   * By default, 'rush update' passes --no-prefer-frozen-lockfile to 'pnpm install'.
+   * Set this option to true to pass '--prefer-frozen-lockfile' instead to minimize shrinkwrap changes.
+   */
+  "usePnpmPreferFrozenLockfileForRushUpdate": true,
```

Comment:
Unrelated?

Reply:
I turned this on because it was bothering me that rush didn't think anything changed when I dropped the dependency. That was probably actually a rush bug, though. I'll exclude this, it should be its own PR.